### PR TITLE
Add runbook.md

### PR DESCRIPTION
**Pull request by fenglish**

### CODE REVIEW

File: runbook.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+You can read about how to rotate an AWS key [over on the Customer Products Wiki](https://customer-products.in.ft.com/wiki/Rotating-AWS-Keys)
+See the Customer Products [key management and troubleshooting wiki page](https://customer-products.in.ft.com/wiki/Key-Management-and-Troubleshooting)
+
+## More Information
+
+n-ui release failure causes n-ui parity issues for anyone trying to build a user-facing app.
+
+There are no monitoring services set up for n-ui. Services which depend on it are individually monitored.
+
+Assets generated by n-ui are stored in a [hashed assets](https://biz-ops.in.ft.com/System/next-hashedasset) bucket which is monitored as Platinum service.
```
Comment (sjparkinson) [inline runbook.md]: This should also be declared as a relationship with the next-hashedasset system under the `Dependencies` section.

File: runbook.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+‍https://github.com/Financial-Times/n-ui
```
Comment (sjparkinson) [inline runbook.md]: This will be available under https://runbooks.in.ft.com/next-n-ui#technical-overview automatically.

```suggestion
```

**Comment by sjparkinson** [comment]
@pixelandpage and @fenglish, do try and make some time for this PR. Let me know if it'd be helpful to sit down together to run through the suggestions.

**Comment by pixelandpage** [comment]
Thanks for the review @sjparkinson, I had a bit of bother getting runbook.md to parse the Dependencies section, but this should be looking a bit better now.

**Pull request merged**